mobile/src/modules/transactions/screens/ScanQRCodeScreen.tsx

```diff
@@ -16,13 +17,17 @@ import { transactions } from '../translations';
 
 class ScanQRCodeScreenComponent extends React.PureComponent<NavigationInjectedProps> {
   onSuccess = (e: Event) => {
-    if (e.data.indexOf('requestBurst') === -1) {
+    if (e.data.indexOf('requestBurst') === -1 && !isBurstAddress(e.data)) {
```

Comment:
that's all? good one! (I would destructure or push `let url` to the top of this method)